sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -75,7 +75,7 @@ def _create_attribute(self):
         self._error_policy = _ServiceBusErrorPolicy(
             max_retries=self._config.retry_total
         )
-        self._name = "SBSender-{}".format(uuid.uuid4())
+        self._name = kwargs.get("client_identifier","SBSender-{}".format(uuid.uuid4()))
```

Comment:
In the Receiver - we're passing this into `link_properties` in `_create_handler` - but I don't see that happening in the Sender?

Reply:
This property on the link is only for the Receiver and not the Sender. Its some sort of left over from the past iterations.